confgenerator/testdata/valid/linux/metrics-receiver_jvm/golden_fluent_bit_main.conf

```diff
@@ -0,0 +1,94 @@
+@SET buffers_dir=/var/lib/google-cloud-ops-agent/fluent-bit/buffers
```

Comment:
Some of the Fluent Bit golden files changed as part of PR 164. We'll need to rebase / merge `master` HEAD and rerun `-update_golden` in this PR in order to get unit tests to pass:

```
go test -mod=mod github.com/GoogleCloudPlatform/ops-agent/confgenerator -update_golden -v
```